mono/mini/aot-runtime.c

```diff
@@ -2507,10 +2510,35 @@ load_aot_module (MonoAssemblyLoadContext *alc, MonoAssembly *assembly, gpointer
 
 	init_amodule_got (amodule, TRUE);
 
+#ifdef HOST_WASM
+	/*
+	 * Emscripten could reorder code so methods from different aot images are intermixed. Compute the continuous
+	 * regions of code and register each of them separately.
+	 */
+	sort_methods (amodule);
+	gpointer start = NULL;
+	for (i = 0; i < amodule->sorted_methods_len; ++i) {
+		//printf ("%s %d\n", amodule->assembly->aname.name, amodule->sorted_methods [i]);
+	}
```

Comment:
nit: debug loop still runs every time

```suggestion
#if 0
	for (i = 0; i < amodule->sorted_methods_len; ++i) {
		printf ("%s %d\n", amodule->assembly->aname.name, amodule->sorted_methods [i]);
	}
#endif
```